changes/440.feature.rst

```diff
@@ -0,0 +1 @@
+Change default compression to lz4.
```

Comment:
Maybe expand this to indicate that compression by default starts here?

Reply:
Thanks! How does the updated changelog in [1ff39c2](https://github.com/spacetelescope/roman_datamodels/pull/440/commits/1ff39c2616339adb8d98130e85bb25ba56ba4ae9) look?
```
Change default compression to lz4. The previous default was no compression.
```

Reply:
Looks good.